src/main/io/gps.c

```diff
@@ -96,6 +96,7 @@ uint8_t GPS_svinfo_cno[GPS_SV_MAXSATS_M8N];
 
 static serialPort_t *gpsPort;
 static float gpsDataIntervalSeconds;
+static bool newDataForPosHold = false;
```

Comment:
not needed. we have a function that runs when new GPS data is available.
```suggestion
```

Reply:
done